cmake/compilers/Clang.cmake

```diff
@@ -27,7 +27,7 @@ elseif (MSVC)
 else()
     set(TBB_LINK_DEF_FILE_FLAG -Wl,--version-script=)
     set(TBB_DEF_FILE_PREFIX lin${TBB_ARCH})
-    set(TBB_TEST_COMPILE_FLAGS ${TBB_TEST_COMPILE_FLAGS} -ffp-model=precise)
+    set(TBB_TEST_COMPILE_FLAGS ${TBB_TEST_COMPILE_FLAGS} $<$<NOT:$<VERSION_LESS:${CMAKE_CXX_COMPILER_VERSION},10.0>>:-ffp-model=precise)
```

Comment:
Why not ``$<CXX_COMPILER_VERSION>`` instead of ``${CMAKE_CXX_COMPILER_VERSION}``?

https://cmake.org/cmake/help/v3.1/manual/cmake-generator-expressions.7.html#informational-expressions

Reply:
We use `$<CXX_COMPILER_VERSION>` in this file already.